Presentation.Web/app/services/dataProcessingAgreementService.ts

```diff
@@ -1,13 +1,27 @@
 ﻿module Kitos.Services.DataProcessing {
     export interface IDataProcessingAgreementService {
         create(organizationId: number, name: string): angular.IPromise<IDataProcessingAgreementCreatedResult>;
-        //TODO: Extend with type safe methods for getting and changing data
+        delete(dataProcessingAgreementId: number): angular.IPromise<IDataProcessingAgreementDeletedResult>;
```

Comment:
du mangler en GET her (se tidligere kommentar) og den skal jo så returnere en typestærk version af dba så du skal også lige have defineret sådan en. Det er heldigvis nemt for der er jo kun 2 properties - Id og Name